examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -202,7 +217,16 @@
 
         params.productAttestationAuthorityCertificates = productAttestationAuthorityCertificates;
 
-        __auto_type * controller = [[MTRDeviceController alloc] initWithParameters:params error:&error];
+        MTRDeviceController * controller = nil;
+        if (mUseXPC.ValueOr(false) || mUseXPCServiceName.HasValue()) {
+            __auto_type * identifier = uuidString;
+            if (commissionerName.compare(identities[i]) == 0 && mUseXPCControllerId.HasValue()) {
```

Comment:
I don't understand this logic.  Does this only allow doing XPC for a single identity?

At the very least, this needs documentation explaining what's going on here.

Reply:
I have removed `use-xpc-controller-id` for the moment. I plan to add it back in a followup in a more understandable implementation.